docs/4._LLM_Providers/openai.md

```diff
@@ -0,0 +1,17 @@
+# Open AI
+
+Setup the API key in your `.env` file:
+
+```bash
+OPENAI_API_KEY=your-api-key
+```
+
+!!! note "Switching from OpenAI to Azure OpenAI Services"
+    This is made easy by changing the `MAIN_MODEL_API_TYPE` in your `.env` file.
+
+    
```

Comment:
Maybe we should also mention "backup engine" here.

Reply:
done